run/GCHP/ExtData.rc.templates/ExtData.rc.carbon

```diff
@@ -229,194 +241,224 @@ CAN_SOLID_WASTE_COAST        kg/m2/s N Y - none none  solid_waste_total
+EDGAR7_CH4_OTHER__4F            kg/m2/s N Y F%y4-%m2-01T00:00:00  none none  emi_ch4  ./HcoDir/CH4/v2024-01/EDGARv7/%y4/v7.0_FT2021_CH4_%y4_AWB.0.1x0.1.nc
+EDGAR7_CH4_OTHER__6C            kg/m2/s N Y F%y4-%m2-01T00:00:00  none none  emi_ch4  ./HcoDir/CH4/v2024-01/EDGARv7/%y4/v7.0_FT2021_CH4_%y4_SWD_INC.0.1x0.1.nc
+
+
+# --- JPL WetCHARTs v1.0 (Bloom et al., https://doi.org/10.3334/ORNLDAAC/1502) ---
+#
+# Use updated files (v2024-01); these are COARDS-compliant.
+#
+# Comment this when "--> JPL_WETCHARTS: false" is in HEMCO_Config.rc.
+JPLW_CH4  kg/m2/s N Y F%y4-%m2-01T00:00:00 none none  emi_ch4  ./HcoDir/CH4/v2024-01/JPL_WetCharts/HEensemble/JPL_WetCharts_2010-2019.Ensemble_Mean.0.5x0.5.nc
+
 
 # --- Geological seeps ---
+#
+# Comment this when "--> SEEPS : false" is in HEMCO_Config.rc.
 CH4_SEEPS kg/m2/s N Y - none none  emi_ch4  ./HcoDir/CH4/v2020-04/Seeps/Etiope_CH4GeologicalEmis_ScaledToHmiel.1x1.nc
 
-# --- Emissions from lakes (Maasakkers et al 2019) ---
-# --- NOTE: Superseded by WetCHARTs            ---
-#CH4_LAKES kg/m2/s N Y F%y4-%m2-01T00:00:00 none none  emi_ch4  ./HcoDir/CH4/v2022-11/Lakes/Maasakkers_Lakes_%y4.01x01.nc
 
 # --- Emissions from reservoirs (Delwich et al 2022) ---
-CH4_RES_DAM kg/m2/s N Y F%y4-%m2-01T00:00:00 none none  CH4emis  ./HcoDir/CH4/v2023-04/ResME/ResME_Potential_Dam_Emissions.0.1x0.1.nc
-CH4_RES_SFC kg/m2/s N Y F%y4-%m2-01T00:00:00 none none  CH4emis  ./HcoDir/CH4/v2023-04/ResME/ResME_Surface_Emissions.0.1x0.1.nc
+#
+# Use updated files (v2024-01); these are COARDS-compliant.
+#
+# Comment these when "--> RESERVOIRS : false" is in HEMCO_Config.rc.
+CH4_RES_DAM kg/m2/s N Y - none none CH4emis  ./HcoDir/CH4/v2024-01/ResME/ResME_Dam_Emissions.0.1x0.1.nc
+CH4_RES_SFC kg/m2/s N Y - none none CH4emis  ./HcoDir/CH4/v2024-01/ResME/ResME_Surface_Emissions.0.1x0.1.nc
+
 
 # --- Emissions from termites (Fung et al 1991) ---
+#
+# Comment this when "--> FUNG_TERMITES : false" is in HEMCO_Config.rc.
 CH4_TERMITES    kg/m2/s N Y - none none  CH4  ./HcoDir/CH4/v2022-11/4x5/termites.geos.4x5.nc
 
-# --- Soil absoprtion (Fung et al 1991) ---
-# --- NOTE: Superseded by MeMo          ---
-#CH4_SOILABSORB  kg/m2/s N Y F%y4-%m2-01T00:00:00 none none  CH4  ./HcoDir/CH4/v2019-10/Fung_SoilAbs/Soil_Absorption_4x5_%y4.nc
 
 # --- Soil absorption fro MeMo model (Murguia-Flores et al. 2018, GMD) ---
-# --- Note; Use climatology by default                                 ---
+#
+# Note; Use climatology by default
+# Comment this when "--> MeMo_SOIL_ABSORPTION :  false" is in HEMCO_Config.rc.
 CH4_SOILABSORB kg/m2/s  2009 Y F%y4-%m2-01T00:00:00 none none  CH4uptake  ./HcoDir/CH4/v2019-10/MeMo_SoilAbs/MeMo_CH4uptake_Climatology.nc
+#
+# Annual emissions are disabled by default
 #CH4_SOILABSORB kg/m2/s N    Y F%y4-%m2-01T00:00:00 none none  CH4uptake  ./HcoDir/CH4/v2019-10/MeMo_SoilAbs/MeMo_CH4uptake_1990-2009.nc
 
+# --- BBMIPs ---
+# Add entries here
```

Comment:
Why aren't the entries already here?

Reply:
@lizziel @msulprizio: We could add them but that might make the ExtData.rc too large.  I was going by the ExtData.rc.fullchem example where we don't include lines that are commented out.  We can rethink this though.

Reply:
I don't think the file length is an issue anymore since ESMF increased the maximum per request from GMAO.

Reply:
I think it may be best to just remove these lines in ExtData.rc. People can add them if they want, but there is no need for them to be there by default. Most users will not use these options.

Reply:
These comments have been removed in https://github.com/geoschem/geos-chem/pull/1992/commits/e837aba3ec9d112b4ab4cc501707e621e43db042